tests/test_generate.py

```diff
@@ -1,3 +1,4 @@
+from seisbench import data
```

Comment:
Looks like an unused import. Could you check and delete if it's indeed unused?

Reply:
Well spotted, thanks! Removed.